windows/installer-source/KaliteSetupScript.iss

```diff
@@ -196,42 +193,20 @@ begin
     end;
 end;
 
-procedure Backup013Files;
+{ Copy files a la the gitmigrate management command, but use native windows executables instead. }
+{ We assume the installing user is the main user of the app, so content goes into that user's .kalite dir}
+procedure DoGitMigrate;
 var
-    prevDBDir, prevContentDir: String;
-    resCode: integer;
+    retCodeContent, retCodeDB, retCode : integer;
 begin
-    { Copy the old database and content folders, preserving the dir structure, then after installing then new files... }
-    { ...run gitmigrate management command and delete the old stuff. }
-    prevAppBackupDir := ExpandConstant('{tmp}') + '\prev-ka-lite';  // Expand this when we use it, instead of earlier.
-    prevDBDir := WizardForm.PrevAppDir + '\ka-lite\kalite\database';
-    prevContentDir := WizardForm.PrevAppDir + '\ka-lite\content';
-
-    if Not ForceDirectories(prevAppBackupDir + '\kalite\') then
-    begin
-        MsgBox('Fatal error' #13#13 'Failed to create backup directories at: ' + prevAppBackupDir, mbError, MB_OK);
-        forceCancel := True;
-        WizardForm.Close;
-    end;
-
-    MsgBox('Setup will now copy your prior user data for migration. If you have a lot of data, this may take some time!', mbInformation, MB_OK);
-
-    Exec(ExpandConstant('{cmd}'), '/S /C "xcopy "' + prevDBDir + '" "' + prevAppBackupDir + '\kalite\database\" /E "', '', SW_HIDE, ewWaitUntilTerminated, resCode);
-    if resCode <> 0 then
-    begin
-        MsgBox('Fatal error' #13#13 'Failed to backup database directory.', mbError, MB_OK);
-        forceCancel := True;
-        WizardForm.Close;
-    end;
-    Exec(ExpandConstant('{cmd}'), '/S /C "xcopy "' + prevContentDir + '" "' + prevAppBackupDir + '\content\" /E"', '', SW_HIDE, ewWaitUntilTerminated, resCode);
-    if resCode <> 0 then
+    MsgBox('Migrating old data to current user''s %USERPROFILE%\.kalite\ directory.', mbInformation, MB_OK);
+    Exec(ExpandConstant('{cmd}'), '/S /C "mkdir "%USERPROFILE%\.kalite""', '', SW_SHOW, ewWaitUntilTerminated, retCode);
+    Exec(ExpandConstant('{cmd}'), '/S /C "xcopy "' + WizardForm.PrevAppDir + '\ka-lite\content" "%USERPROFILE%\.kalite\content\" /Y"', '', SW_SHOW, ewWaitUntilTerminated, retCodeContent);

```

Comment:
These commands have been modified slightly -- instead of recursively copying the entire source directory (which would include assessment items) we just do a shallow copy of the old content directory. All the videos live (flat) in that directory, which is what we wish to copy.